CMakeLists.txt

```diff
@@ -1,9 +1,11 @@
 cmake_minimum_required(VERSION 3.10)
 project("flutter_linuxes" LANGUAGES CXX C)
 
-# Rules of compiler and cpp version.
-set(CMAKE_C_COMPILER clang)
-set(CMAKE_CXX_COMPILER clang++)
+if(NOT DEFINED CMAKE_TOOLCHAIN_FILE)
+    set(CMAKE_C_COMPILER clang)
+    set(CMAKE_CXX_COMPILER clang++)
+endif()
+# cpp version.
```

Comment:
Please delete this comment. It's very clear. Rather, you need to add the comment you will fail if you try to build with gcc compiler instead of clang.